docs/integrations/common-workflow-steps/build-cache/how-to-share-file-between-build-profiles.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to Share Files between Build Profiles 
+metaTitle: How to Share Files between Build Profiles
+metaDescription: How to Share Files between Build Profiles
+sidebar_position: 4
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+
+
+# How to Share Files between Build Profiles
+
+With the build cache structure offered by Appcircle, you can share your cache files between different [**Build Profiles**](https://docs.appcircle.io/build/adding-a-build-profile/). This file sharing will allow you to generate your packages faster in different Build Profiles and reduce your build time. Below is a simple example of how you can do this step by step. 
+
+:::info
+This simple example will use our CocoaPods files in a different build profiles. If you want to use a cache other than dependencies, please refer to the documentation for the [**Cache Push**](https://docs.appcircle.io/workflows/common-workflow-steps/#cache-push) component.
+:::
+
+:::caution
+In order to share cache between Build Profiles, the [**Cache Pull**](https://docs.appcircle.io/workflows/common-workflow-steps/#cache-pull) component must be added to the related pipeline.
+:::
+
+- First of all, we start by caching our CocoaPods files in the **`Appcircle Team`** build profile. We will use these files in our **`Appcircle Team 2`** build profile later. For this, the Cache Push step must be added to the workflow after the CocoaPods Install step in the first build profile.
```

Comment:
Can you add image for Cache Push input, especially for Cache label

Reply:
for sharing file between pipeline or build profiles, cache label of cache push step does not necessary. It will be generated with env variables. And It will be as default.